tests/integration/components/gh-image-uploader-with-preview-test.js

```diff
@@ -1,46 +1,41 @@
 /* jshint expr:true */
 import {expect} from 'chai';
-import {
-    describeComponent,
-    it
-} from 'ember-mocha';
+import {describe, it} from 'mocha';
+import {setupComponentTest} from 'ember-mocha';
 import hbs from 'htmlbars-inline-precompile';
 import run from 'ember-runloop';
 import sinon from 'sinon';
 
-describeComponent(
-    'gh-image-uploader-with-preview',
-    'Integration: Component: gh-image-uploader-with-preview',
-    {
+describe('Integration: Component: gh-image-uploader-with-preview', function() {
+    setupComponentTest('gh-image-uploader-with-preview', {
         integration: true
-    },
-    function() {
-        it('renders image if provided', function() {
-            this.set('image', 'http://example.com/test.png');
+    });
 
-            this.render(hbs`{{gh-image-uploader-with-preview image=image}}`);
+    it('renders image if provided', function() {
+        this.set('image', 'http://example.com/test.png');
 
-            expect(this.$('.gh-image-uploader.-with-image').length).to.equal(1);
-            expect(this.$('img').attr('src')).to.equal('http://example.com/test.png');
-        });
+        this.render(hbs`{{gh-image-uploader-with-preview image=image}}`);
 
-        it('renders upload form when no image provided', function () {
-            this.render(hbs`{{gh-image-uploader-with-preview image=image}}`);
+        expect(this.$('.gh-image-uploader.-with-image').length).to.equal(1);
+        expect(this.$('img').attr('src')).to.equal('http://example.com/test.png');
+    });
 
-            expect(this.$('input[type="file"]').length).to.equal(1);
-        });
+    it('renders upload form when no image provided', function () {
+        this.render(hbs`{{gh-image-uploader-with-preview image=image}}`);
 
-        it('triggers remove action when delete icon is clicked', function () {
-            let remove = sinon.spy();
-            this.set('remove', remove);
-            this.set('image', 'http://example.com/test.png');
+        expect(this.$('input[type="file"]').length).to.equal(1);
```

Comment:
@acburdine since @rwjblue doesn't agree with me in https://github.com/emberjs/ember-mocha/issues/110 I've changed the test code here to explicitly reset the `image` property before running the test. This should get you going while we figure out what the intended behavior is supposed to be.